static/app/views/issueDetails/header.spec.tsx

```diff
@@ -184,20 +187,22 @@ describe('GroupHeader', () => {
           organization={orgWithFeatures}
           project={projectWithSimilarityView}
         />,
-        {organization: orgWithFeatures}
+        {organization: orgWithFeatures, router}
       );
 
       await userEvent.click(screen.getByRole('tab', {name: /details/i}));
-      expect(browserHistory.push).toHaveBeenLastCalledWith('BASE_URL/');
+      expect(router.push).toHaveBeenLastCalledWith(
+        expect.objectContaining({pathname: 'BASE_URL/'})
+      );
```

Comment:
@scttcper This string->object is the only refactor that raises an eyebrow for me, can you confirm this is expected.